docs/content/integrations/airbyte-cloud.mdx

```diff
@@ -0,0 +1,259 @@
+---
+title: "Using Dagster with Airbyte Cloud"
+description: Integrate your Airbyte Cloud connections into Dagster.
+---
+
+# Using Airbyte Cloud with Dagster
+
+<Note>
+  Using self-hosted Airbyte? Check out the{" "}
+  <a href="/integrations/airbyte">
+    guide on using self-hosted Airbyte with Dagster.
+  </a>
+</Note>
+
+Dagster can orchestrate your Airbyte Cloud connections, making it easy to chain an Airbyte sync with upstream or downstream steps in your workflow.
+
+This guide focuses on how to work with Airbyte connections using Dagster's [software-defined asset (SDA)](/concepts/assets/software-defined-assets) framework.
```

Comment:
```suggestion
This guide focuses on how to work with Airbyte Cloud connections using Dagster's [software-defined asset (SDA)](/concepts/assets/software-defined-assets) framework.
```

Should this be `Airbyte Cloud`? There are some places in this guide that use just `Airbyte` vs `Airbyte Cloud` - we should be consistent unless there's a technical reason to just use `Airbyte`